src/routes/AuthRoutes.tsx

```
@@ -21,6 +24,17 @@ const routes = [
 		path: '/explorebenefits',
 		component: ExploreBenefits,
 	},
+	{
+		path: '/vcscans',
+		component: ScanVC,
+	},
+	{
+		path: '/document-scanner',
+		component: () => {
+			const { userData } = useContext(AuthContext)!;
```

ESLint

[error] React Hook "useContext" is called in function "component" that is neither a React function component nor a custom React Hook function. React component names must start with an uppercase letter. React Hook names must start with the word "use".

(react-hooks/rules-of-hooks)